lighthouse-core/test/config/config-plugin-test.js

```diff
@@ -84,6 +93,7 @@ describe('ConfigPlugin', () => {
       categories: {
         'lighthouse-plugin-evil': evilCategory,
       },
+      groups: undefined,
```

Comment:
do these two additions add anything to the test? My brain may be just foggy today and I'm missing it :)

Reply:
Tests were failing without this because the undefined group object is not yet stripped at this point.